### PR TITLE
[VEN-2937]: grant permissions to timelock on optimism and zksync

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

### CODE REVIEW

File: deploy/008-configure-acm-commands-aggregator.ts [hunk 1]
```diff
@@ -302,6 +416,36 @@ const revokePermissions: Permissions = {
     ...getVTokenRevokePermissions(OPBNBMAINNET_GUARDIAN),
     ...getXVSBridgeAdminRevokePermissions(OPBNBMAINNET_XVS_BRIDGE_ADMIN, OPBNBMAINNET_GUARDIAN),
   ],
+  opmainnet: [
+    ...getPrimeRevokePermissions(OPMAINNET_PRIME, OPMAINNET_GUARDIAN),
+    ...getPrimeLiquidityProviderRevokePermissions(OPMAINNET_PLP, OPMAINNET_GUARDIAN),
+    ...getResilientOracleRevokePermissions(OPMAINNET_RESILIENT_ORACLE, OPMAINNET_GUARDIAN),
+    ...getBoundValidatorRevokePermissions(OPMAINNET_BOUND_VALIDATOR, OPMAINNET_GUARDIAN),
+    ...getXVSVaultRevokePermissions(OPMAINNET_XVS, OPMAINNET_GUARDIAN),
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply deploy/008-configure-acm-commands-aggregator.ts]: I’ve fixed these addresses on zkSync and Optimism (commit: [3eddebd](https://github.com/VenusProtocol/governance-contracts/pull/100/commits/3eddebd76276d5a06a647f5ddb79310a5908934b)).
Additionally, the address on the previous network also needed fixing, which I’ve completed (commit: [4d5f878](https://github.com/VenusProtocol/governance-contracts/pull/100/commits/4d5f878f7d2050e33f2cbe298964741457ac085f)).

@chechu, we’ll also need a VIP to implement these changes on the previous network. WDYT?"

[the remark this answers is not in the document]
Comment (chechu) [reply deploy/008-configure-acm-commands-aggregator.ts]: I agree. We could add the fixes to the same VIP where the permissions for OP and zkSync will be configured (https://github.com/VenusProtocol/vips/pull/431)

[the remark this answers is not in the document]
Comment (chechu) [reply deploy/008-configure-acm-commands-aggregator.ts]: We could pre-load the fixes in the ACMCommandsAggregator contract, and invoke the remote command in the VIP

File: deploy/008-configure-acm-commands-aggregator.ts [hunk 2]
```diff
@@ -295,17 +409,48 @@ const revokePermissions: Permissions = {
...
   ],
+  opmainnet: [
+    ...getPrimeRevokePermissions(OPMAINNET_PRIME, OPMAINNET_GUARDIAN),
+    ...getPrimeLiquidityProviderRevokePermissions(OPMAINNET_PLP, OPMAINNET_GUARDIAN),
+    ...getResilientOracleRevokePermissions(OPMAINNET_RESILIENT_ORACLE, OPMAINNET_GUARDIAN),
+    ...getBoundValidatorRevokePermissions(OPMAINNET_BOUND_VALIDATOR, OPMAINNET_GUARDIAN),
+    ...getXVSVaultRevokePermissions(OPMAINNET_XVS_VAULT_PROXY, OPMAINNET_GUARDIAN),
+    ...getRewardDistributorRevokePermissions(OPMAINNET_GUARDIAN, true),
+    ...getIRMRevokePermissions(OPMAINNET_GUARDIAN),
+    ...getPoolRegistryRevokePermissions(OPSEPOLIA_POOL_REGISTRY, OPSEPOLIA_GUARDIAN),
```
Comment (chechu) [inline deploy/008-configure-acm-commands-aggregator.ts:427]:
```suggestion
    ...getPoolRegistryRevokePermissions(OPMAINNET_POOL_REGISTRY, OPMAINNET_GUARDIAN),
```

**Review by chechu — APPROVED** [review]

**Pull request merged**